app/grandchallenge/profiles/migrations/0015_alter_userprofile_receive_notification_emails.py

```diff
@@ -0,0 +1,40 @@
+# Generated by Django 4.2.11 on 2024-03-13 08:57
+
+from django.db import migrations, models
+
+from grandchallenge.profiles.models import NotificationSubscriptionOptions
+
+
+def migrate_notification_email_preferences(apps, _schema_editor):
+    UserProfile = apps.get_model("profiles", "UserProfile")  # noqa: N806
+    UserProfile.objects.filter(receive_notification_emails="true").update(
+        receive_notification_emails=NotificationSubscriptionOptions.DAILY_SUMMARY
+    )
+    UserProfile.objects.filter(receive_notification_emails="false").update(
+        receive_notification_emails=NotificationSubscriptionOptions.DISABLED
+    )
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("profiles", "0014_userprofile_unread_messages_email_last_sent_at"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="userprofile",
+            name="receive_notification_emails",
+            field=models.CharField(
+                choices=[
+                    ("DAILY_SUMMARY", "Send me an email with a daily summary"),
+                    ("DISABLED", "Do not send me notification emails"),
+                    ("INSTANT", "Send me an email immediately"),
+                ],
+                default="DAILY_SUMMARY",
+                help_text="Whether to receive emails about unread notifications and how often (immediately vs. once a day if necessary).",
+                max_length=50,
+            ),
+        ),
+        migrations.RunPython(migrate_notification_email_preferences),
```

Comment:
This should be marked as elidable as there is nothing to do with an empty db so can be removed when we squash the migrations.

Reply:
Ah true, good catch.